dev-python/django-prometheus/django-prometheus-2.1.0.ebuild

```diff
@@ -15,7 +15,6 @@ SLOT="0"
 KEYWORDS="~amd64"
 
 DEPEND=">=dev-python/prometheus_client-0.7[${PYTHON_USEDEP}]"
-RDEPEND="${DEPEND}"
```

Comment:
I might be mistaken, but I think we want here to define `RDEPEND` and leave `DEPEND` empty

Reply:
yes

Reply:
yep, I'll fix it